src/reflect/scala/reflect/internal/Reporting.scala

```diff
@@ -45,28 +45,18 @@ trait Reporting { self : Positions =>
 
   def supplementErrorMessage(errorMessage: String) = currentRun.reporting.supplementErrorMessage(errorMessage)
 
-  @deprecatedOverriding("This forwards to the corresponding method in reporter -- override reporter instead", "2.11.2")
```

Comment:
I would prefer to leave these in place and make them plain `@deprecated`. Sadly, you never know who may be calling us here.